zkevm-circuits/src/evm_circuit/util/common_gadget.rs

```diff
@@ -370,8 +370,8 @@ impl<F: Field, const N_ADDENDS: usize, const INCREASE: bool>
 pub(crate) struct TransferToGadget<F> {
     receiver: UpdateBalanceGadget<F, 2, true>,
```

Comment:
suggest `balance: UpdateBalanceGadget<F, 2, true>` or `receiver_balance: UpdateBalanceGadget<F, 2, true>` for more self-explanation

Reply:
Sounds good. Fixed in [990afe1](https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1777/commits/990afe19f694fa9d4fc037914758be31b7d224d1)